Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN apk update && apk upgrade && \
 RUN ln -s `which nodejs` /usr/bin/node
 
 # install phantomjs
-RUN npm install -g phantomjs-prebuilt
+RUN npm install -g phantomjs-prebuilt@2.1.11
```

Comment:
This is so odd to me. I think in theory you should use 2.1.14 instead of 2.1.11 (may as well use the newest, plus uh, 2.1.11 isn't even a tag on their repo?)